app/scripts/stores/author.js

```diff
@@ -188,12 +191,36 @@ export default Reflux.createStore({
 
   createPostComplete: function(postData) {
     var post = new Post(postData);
-    post.author = new Author(post.author);
+
+    // Cyclic reference
+    post.author = this.currentAuthor;
+    // add new post
     this.currentAuthor.posts.push(post);
-    this.trigger({displayAuthor: this.currentAuthor});
+    // trigger update
+    this.trigger({displayPosts: this.currentAuthor.sortedPosts()});
```

Comment:
We should probably enforce the sorting in the backend. Unfortunately we still need to sort out the date serializer stuff first.